ExampleConfigurations/Delta/Configuration.h

```diff
@@ -17,8 +17,8 @@
 // User-specified version info of this build to display in [Pronterface, etc] terminal window during
 // startup. Implementation of an idea by Prof Braino to inform user that any changes made to this
 // build by the user have been successfully uploaded into firmware.
-#define STRING_VERSION_CONFIG_H __DATE__ " " __TIME__ // build date and time
-#define STRING_CONFIG_H_AUTHOR "(none, default config)" // Who made the changes.
+#define STRING_BUILD_DATE __DATE__ " " __TIME__ // build date and time
+#define STRING_AUTHOR "(none, default config)" // Who made the changes.
```

Comment:
This would need to match the changes made in the default configuration.h file as the strings are built differently now after the changes to marlin_main.cpp, same goes for the other example configurations.